### ISSUE TITLE
Add Support partner option

### ISSUE THREAD

**Issue by tommykr2** [MicrosoftDocs/OfficeDocs-SharePoint#2535]
The provided workaround in the new experience is not sufficient. It says to use powershell BUT says "go to a site where you've added the support partner and copy the encoded string". Well, what if you haven't already added a support partner to any sites in this tenant? There is no replacement for this functionality which was ripped away when the classic site collection page was discontinued. Very frustrating

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 659548ea-eb6e-8eee-2892-f352512db6a1
* Version Independent ID: 2e0e6017-2b29-e831-ff61-0a798dfb175f
* Content: [Manage site admins - SharePoint in Microsoft 365](https://docs.microsoft.com/en-us/sharepoint/manage-site-collection-administrators)
* Content Source: [SharePoint/SharePointOnline/manage-site-collection-administrators.md](https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/blob/live/SharePoint/SharePointOnline/manage-site-collection-administrators.md)
* Service: **sharepoint-online**
* GitHub Login: @kaarins
* Microsoft Alias: **kaarins**

**Comment by scanum** [comment]
@tommykr2 Thank you for submitting feedback and contributing to the docs. We are currently investigating this.

**Comment by tommykr2** [comment]
> @tommykr2 Thank you for submitting feedback and contributing to the docs. We are currently investigating this.

@scanum Thanks! I can provide lots more detail if my first post is not clear enough.

**Comment by get-itips** [comment]
Hello @tommykr2 how are you?
Can not find that on the article you created the issue for (https://docs.microsoft.com/en-us/sharepoint/manage-site-collection-administrators).
I think you refer to this article instead https://docs.microsoft.com/en-us/sharepoint/site-collections-page

@kaarins Kaarin can you help us confirm what to do if you do not have a site where you already added a support partner? I appreciate it.

**Comment by tommykr2** [comment]
@get-itips yes you're right. I meant the second article you linked to. I must have had two tabs open... sorry about that!

When we get new customers we used to use the classic site collection admin page to "add support partner" to any classic site. This gives our delegated account access to that site. From there we could follow the directions in that article - copy encoded string out of that site and apply to all other site collections via powershell. However, now that the classic site collection admin page. we have no method to obtain the encoded string. This seems silly - there must be a way to obtain this via PowerShell or other.
I opened a support case with MSFT and they told me "sorry, open a uservoice topic". Meanwhile, we're stuck without the ability to use delegated accounts to access more and more client SPO sites.
